api/.env

```diff
@@ -74,3 +74,10 @@ SENTRY_API_DSN=""
 
 FRONTEND_BASE_URL="http://localhost:3000"
 MAIL_FROM="info@ecamp3.ch"
+
+###> google/recaptcha ###
+# To use Google Recaptcha, you must register a site on Recaptcha's admin panel:
+# https://www.google.com/recaptcha/admin
+GOOGLE_RECAPTCHA_SITE_KEY=6LdiBj8fAAAAAG8sG_WhT6TMTtGO2iSSk1y6oCPd
```

Comment:
Das würde ich jetzt im Gegensatz zu den oauth test credentials nicht in git einchecken, sondern recaptcha optional machen und im helm chart ergänzen.